chemie/rentalservice/templates/rentalservice/detail.html

```diff
@@ -63,15 +63,17 @@ <h5>Beskrivelse:</h5>
 
 
```

Comment:
The image on this page is very large. Could perhaps be scaled down, so that the user sees the information below the image without scrolling